cmd/snap-repair/runner_test.go

```diff
@@ -1391,3 +1415,256 @@ AXNpZw==`}
 	c.Assert(err, IsNil)
 	c.Check(string(scrpt), Equals, "exit 0\n")
 }
+
+func makeMockRepair(script string) string {
+	return fmt.Sprintf(`type: repair
+authority-id: canonical
+brand-id: canonical
+repair-id: 1
+series:
+  - 16
+timestamp: 2017-07-02T12:00:00Z
+body-length: %d
+sign-key-sha3-384: KPIl7M4vQ9d4AUjkoU41TGAwtOMLc_bWUCeW8AvdRWD4_xcP60Oo4ABsFNo6BtXj
+
+%s
+
+AXNpZw==`, len(script), script)
```

Comment:
What is this tail here? I understand this is an assertion and here we'd have the signature? Am I missing something?